start.sh

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env bash
 
-trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+trap "trap - SIGTERM && kill -- -$$" SIGTERM EXIT
+trap 'kill $(jobs -p)' SIGINT
```

Comment:
When the script is sourced, the nodes started in background can be killed easily with ctrl-c.